### PR TITLE
Show ledger addrs on key list

### PR DESCRIPTION
**Pull request by felipemadero**
Closes https://github.com/ava-labs/avalanche-cli/issues/365

- Add --ledger flag that receives a list of indices and shows information for those, avoiding providing information for stored keys
- Add flags to control networks and chains: --local, --testnet, --fuji, --mainnet, --cchain
- Refactor of all code

### CODE REVIEW

File: cmd/keycmd/list.go [hunk 1]
```diff
@@ -24,107 +26,351 @@ import (
...
+			return []addressInfo{}, err
+		}
+		addrInfos = append(addrInfos, ledgerAddrInfos...)
+	}
+	return addrInfos, nil
+}
 
-	if allNetworks {
-		supportedNetworks[models.Local.String()] = 0
+func getLedgerAddrInfo(
```
Comment (dasconnor) [inline cmd/keycmd/list.go:278]: I'm not sure what the difference between this function and `getLedgerAddrInfos` is. The difference isn't clear from the name.

**Review by dasconnor — COMMENTED** [review]
I think the code looks good, but could you include a video or some screenshots? It's a little hard for me to visualize what everything here looks like to the end user.

**Comment by felipemadero** [comment]
> I think the code looks good, but could you include a video or some screenshots? It's a little hard for me to visualize what everything here looks like to the end user.

https://asciinema.org/a/537864

File: cmd/keycmd/list.go (hunk 1 above)
Comment (felipemadero) [reply cmd/keycmd/list.go:278]: renamed to getLedgerIndices info and getLedgerIndexInfo

**Comment by dasconnor** [comment]
That video was great! Very helpful! I do have a couple recommendations. I'm not sure I like `mainnet` being the default network. I think we probably want to prompt for network by default.

Also, I don't like having to set `--mainnet=false`. If no flags, we should prompt. Then any flag can just be additive. So you can do `--mainnet` and `--testnet` and it will be fine.

**Review by dasconnor — APPROVED** [review]

**Pull request merged**